lua/hawtkeys/ts.lua

```diff
@@ -378,7 +378,7 @@ local function get_keymaps_from_lazy()
                     for _, key in pairs(v._.handlers.keys) do
                         local map = {
                             lhs = key.lhs,
-                            rhs = key.rhs,
+                            rhs = tostring(key.rhs),
```

Comment:
could also do a check for function values and check `debug.getinfo`

Reply:
![image](https://github.com/tris203/hawtkeys.nvim/assets/18444302/1d518046-5170-4334-99f3-febd89c987fb)
done that, will print using the file and line where it is defined now